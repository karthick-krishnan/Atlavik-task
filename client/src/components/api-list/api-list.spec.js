import React from 'react';
import renderer from 'react-test-renderer';
import ApiList from './api-list';

it('renders correctly', () => {
  const tree = renderer
    .create(<ApiList />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
