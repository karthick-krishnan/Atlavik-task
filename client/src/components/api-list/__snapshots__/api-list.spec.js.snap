// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-table-wrapper"
>
  <div
    className="ant-spin-nested-loading"
  >
    <div
      className="ant-spin-container"
    >
      <div
        className="ant-table ant-table-empty"
      >
        <div
          className="ant-table-container"
        >
          <div
            className="ant-table-content"
            onScroll={[Function]}
            style={Object {}}
          >
            <table
              style={
                Object {
                  "tableLayout": "auto",
                }
              }
            >
              <colgroup />
              <thead
                className="ant-table-thead"
              >
                <tr>
                  <th
                    className="ant-table-cell ant-table-column-has-sorters"
                    colSpan={null}
                    onClick={[Function]}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-table-column-sorters-with-tooltip"
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                    >
                      <div
                        className="ant-table-column-sorters"
                      >
                        <span>
                          API Name
                        </span>
                        <span
                          className="ant-table-column-sorter ant-table-column-sorter-full"
                        >
                          <span
                            className="ant-table-column-sorter-inner"
                          >
                            <span
                              aria-label="caret-up"
                              className="anticon anticon-caret-up ant-table-column-sorter-up"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="caret-up"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="0 0 1024 1024"
                                width="1em"
                              >
                                <path
                                  d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                />
                              </svg>
                            </span>
                            <span
                              aria-label="caret-down"
                              className="anticon anticon-caret-down ant-table-column-sorter-down"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="caret-down"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="0 0 1024 1024"
                                width="1em"
                              >
                                <path
                                  d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </div>
                    </div>
                  </th>
                  <th
                    className="ant-table-cell ant-table-column-has-sorters"
                    colSpan={null}
                    onClick={[Function]}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-table-column-sorters-with-tooltip"
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                    >
                      <div
                        className="ant-table-column-sorters"
                      >
                        <span>
                          Description
                        </span>
                        <span
                          className="ant-table-column-sorter ant-table-column-sorter-full"
                        >
                          <span
                            className="ant-table-column-sorter-inner"
                          >
                            <span
                              aria-label="caret-up"
                              className="anticon anticon-caret-up ant-table-column-sorter-up"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="caret-up"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="0 0 1024 1024"
                                width="1em"
                              >
                                <path
                                  d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                />
                              </svg>
                            </span>
                            <span
                              aria-label="caret-down"
                              className="anticon anticon-caret-down ant-table-column-sorter-down"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="caret-down"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="0 0 1024 1024"
                                width="1em"
                              >
                                <path
                                  d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </div>
                    </div>
                  </th>
                  <th
                    className="ant-table-cell ant-table-column-has-sorters"
                    colSpan={null}
                    onClick={[Function]}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-table-column-sorters-with-tooltip"
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                    >
                      <div
                        className="ant-table-column-sorters"
                      >
                        <span>
                          Category
                        </span>
                        <span
                          className="ant-table-column-sorter ant-table-column-sorter-full"
                        >
                          <span
                            className="ant-table-column-sorter-inner"
                          >
                            <span
                              aria-label="caret-up"
                              className="anticon anticon-caret-up ant-table-column-sorter-up"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="caret-up"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="0 0 1024 1024"
                                width="1em"
                              >
                                <path
                                  d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                />
                              </svg>
                            </span>
                            <span
                              aria-label="caret-down"
                              className="anticon anticon-caret-down ant-table-column-sorter-down"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                className=""
                                data-icon="caret-down"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="0 0 1024 1024"
                                width="1em"
                              >
                                <path
                                  d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </div>
                    </div>
                  </th>
                  <th
                    className="ant-table-cell ant-table-column-has-sorters"
                    colSpan={null}
                    onClick={[Function]}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-table-filter-column"
                    >
                      <span
                        className="ant-table-filter-column-title"
                      >
                        <div
                          className="ant-table-column-sorters-with-tooltip"
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                        >
                          <div
                            className="ant-table-column-sorters"
                          >
                            <span>
                              Auth
                            </span>
                            <span
                              className="ant-table-column-sorter ant-table-column-sorter-full"
                            >
                              <span
                                className="ant-table-column-sorter-inner"
                              >
                                <span
                                  aria-label="caret-up"
                                  className="anticon anticon-caret-up ant-table-column-sorter-up"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    className=""
                                    data-icon="caret-up"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  aria-label="caret-down"
                                  className="anticon anticon-caret-down ant-table-column-sorter-down"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    className=""
                                    data-icon="caret-down"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                    />
                                  </svg>
                                </span>
                              </span>
                            </span>
                          </div>
                        </div>
                      </span>
                      <span
                        className="ant-table-filter-trigger-container"
                        onClick={[Function]}
                      >
                        <span
                          className="ant-dropdown-trigger ant-table-filter-trigger"
                          onClick={[Function]}
                          onMouseDown={[Function]}
                          onTouchStart={[Function]}
                          role="button"
                          tabIndex={-1}
                        >
                          <span
                            aria-label="filter"
                            className="anticon anticon-filter"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              className=""
                              data-icon="filter"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M349 838c0 17.7 14.2 32 31.8 32h262.4c17.6 0 31.8-14.3 31.8-32V642H349v196zm531.1-684H143.9c-24.5 0-39.8 26.7-27.5 48l221.3 376h348.8l221.3-376c12.1-21.3-3.2-48-27.7-48z"
                              />
                            </svg>
                          </span>
                        </span>
                      </span>
                    </div>
                  </th>
                  <th
                    className="ant-table-cell ant-table-column-has-sorters"
                    colSpan={null}
                    onClick={[Function]}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-table-filter-column"
                    >
                      <span
                        className="ant-table-filter-column-title"
                      >
                        <div
                          className="ant-table-column-sorters-with-tooltip"
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                        >
                          <div
                            className="ant-table-column-sorters"
                          >
                            <span>
                              Cors
                            </span>
                            <span
                              className="ant-table-column-sorter ant-table-column-sorter-full"
                            >
                              <span
                                className="ant-table-column-sorter-inner"
                              >
                                <span
                                  aria-label="caret-up"
                                  className="anticon anticon-caret-up ant-table-column-sorter-up"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    className=""
                                    data-icon="caret-up"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  aria-label="caret-down"
                                  className="anticon anticon-caret-down ant-table-column-sorter-down"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    className=""
                                    data-icon="caret-down"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                    />
                                  </svg>
                                </span>
                              </span>
                            </span>
                          </div>
                        </div>
                      </span>
                      <span
                        className="ant-table-filter-trigger-container"
                        onClick={[Function]}
                      >
                        <span
                          className="ant-dropdown-trigger ant-table-filter-trigger"
                          onClick={[Function]}
                          onMouseDown={[Function]}
                          onTouchStart={[Function]}
                          role="button"
                          tabIndex={-1}
                        >
                          <span
                            aria-label="filter"
                            className="anticon anticon-filter"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              className=""
                              data-icon="filter"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M349 838c0 17.7 14.2 32 31.8 32h262.4c17.6 0 31.8-14.3 31.8-32V642H349v196zm531.1-684H143.9c-24.5 0-39.8 26.7-27.5 48l221.3 376h348.8l221.3-376c12.1-21.3-3.2-48-27.7-48z"
                              />
                            </svg>
                          </span>
                        </span>
                      </span>
                    </div>
                  </th>
                  <th
                    className="ant-table-cell ant-table-column-has-sorters"
                    colSpan={null}
                    onClick={[Function]}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-table-filter-column"
                    >
                      <span
                        className="ant-table-filter-column-title"
                      >
                        <div
                          className="ant-table-column-sorters-with-tooltip"
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                        >
                          <div
                            className="ant-table-column-sorters"
                          >
                            <span>
                              HTTPS
                            </span>
                            <span
                              className="ant-table-column-sorter ant-table-column-sorter-full"
                            >
                              <span
                                className="ant-table-column-sorter-inner"
                              >
                                <span
                                  aria-label="caret-up"
                                  className="anticon anticon-caret-up ant-table-column-sorter-up"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    className=""
                                    data-icon="caret-up"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  aria-label="caret-down"
                                  className="anticon anticon-caret-down ant-table-column-sorter-down"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    className=""
                                    data-icon="caret-down"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                    />
                                  </svg>
                                </span>
                              </span>
                            </span>
                          </div>
                        </div>
                      </span>
                      <span
                        className="ant-table-filter-trigger-container"
                        onClick={[Function]}
                      >
                        <span
                          className="ant-dropdown-trigger ant-table-filter-trigger"
                          onClick={[Function]}
                          onMouseDown={[Function]}
                          onTouchStart={[Function]}
                          role="button"
                          tabIndex={-1}
                        >
                          <span
                            aria-label="filter"
                            className="anticon anticon-filter"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              className=""
                              data-icon="filter"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M349 838c0 17.7 14.2 32 31.8 32h262.4c17.6 0 31.8-14.3 31.8-32V642H349v196zm531.1-684H143.9c-24.5 0-39.8 26.7-27.5 48l221.3 376h348.8l221.3-376c12.1-21.3-3.2-48-27.7-48z"
                              />
                            </svg>
                          </span>
                        </span>
                      </span>
                    </div>
                  </th>
                  <th
                    className="ant-table-cell"
                    colSpan={null}
                    rowSpan={null}
                    style={Object {}}
                  >
                    Link
                  </th>
                </tr>
              </thead>
              <tbody
                className="ant-table-tbody"
              >
                <tr
                  className="ant-table-placeholder"
                  style={
                    Object {
                      "display": null,
                    }
                  }
                >
                  <td
                    className="ant-table-cell"
                    colSpan={7}
                    rowSpan={null}
                    style={Object {}}
                  >
                    <div
                      className="ant-empty ant-empty-normal"
                    >
                      <div
                        className="ant-empty-image"
                      >
                        <svg
                          className="ant-empty-img-simple"
                          height="41"
                          viewBox="0 0 64 41"
                          width="64"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <g
                            fill="none"
                            fillRule="evenodd"
                            transform="translate(0 1)"
                          >
                            <ellipse
                              className="ant-empty-img-simple-ellipse"
                              cx="32"
                              cy="33"
                              rx="32"
                              ry="7"
                            />
                            <g
                              className="ant-empty-img-simple-g"
                              fillRule="nonzero"
                            >
                              <path
                                d="M55 12.76L44.854 1.258C44.367.474 43.656 0 42.907 0H21.093c-.749 0-1.46.474-1.947 1.257L9 12.761V22h46v-9.24z"
                              />
                              <path
                                className="ant-empty-img-simple-path"
                                d="M41.613 15.931c0-1.605.994-2.93 2.227-2.931H55v18.137C55 33.26 53.68 35 52.05 35h-40.1C10.32 35 9 33.259 9 31.137V13h11.16c1.233 0 2.227 1.323 2.227 2.928v.022c0 1.605 1.005 2.901 2.237 2.901h14.752c1.232 0 2.237-1.308 2.237-2.913v-.007z"
                              />
                            </g>
                          </g>
                        </svg>
                      </div>
                      <p
                        className="ant-empty-description"
                      >
                        No Data
                      </p>
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
